skaffold.yaml

```diff
@@ -12,20 +12,38 @@ build:
     docker:
       dockerfile: Dockerfile
       buildArgs:
-        AEA_VERSION: "1.14.0"
+        AEA_VERSION: "1.14.0.post1"
         AEA_AGENT: "{{.AEA_AGENT}}"
 
 
 profiles:
-  - name: docs
+  - name: release
     build:
-      tagPolicy:
-        envTemplate:
-          template: "{{.VERSION}}"
-      artifacts:
-      - image: valory/open-autonomy-docs
-        docker:
-          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+        tagPolicy:
+          envTemplate:
+            template: "{{.VERSION}}"
+        artifacts:
+        - image: valory/open-autonomy-open-aea
+          context: deployments/Dockerfiles/open_aea/
+          docker:
+            noCache: true
+            dockerfile: Dockerfile
+            buildArgs:
+              AEA_VERSION: "1.14.0.post1"
+              AEA_AGENT: "open_aea/my_first_aea:0.1.0:bafybeicyirm7y2ygptr5rybwznwof2k6b7fvlsvebsz4pa6cfjijva3rdu"
```

Comment:
This needs to set to some of agent otherwise we cant the right image.

(Make build images is still broken and i didnt fix it as i have no idea what we will be changing the api to)

Reply:
Why do we need to pass an agent here at all. Very messy.

Reply:
Above we use `AEA_AGENT` env var. Why not here?

Reply:
I added this to force the deployment build with a known agent from the docs.